ga/latest/kernel/helpers/runtime/docker-server.sh

```diff
@@ -157,6 +157,12 @@ elif [[ ! -z "$WLP_CHECKPOINT" ]]; then
   # A checkpoint action has been requested; run the checkpoint.sh script.
   checkpoint.sh "$TMP_CHECKPOINT"
 else
+  if [[ ! -z "$SERVICEABILITY_NAMESPACE" ]] && [[ ! -z $HOSTNAME ]]; then
```

Comment:
Can this be set after Infinispan config (line 145), so that InstantOn scenarios can also use this? Any concern with that?

Reply:
That makes sense, I've edited this to include coverage before the InstantOn exec.